backend/src/database/DatabaseSeeder.sql

```diff
@@ -0,0 +1,362 @@
+INSERT INTO Meal_Categories(Name) VALUES
+('Appetizers'),
+('Main Courses'),
+('Desserts'),
+('Drinks');
+
+
+INSERT INTO meals (Name, Description, Price, Image, Meal_Category_Id)
+VALUES
+('Spring Rolls', 'Crispy rolls filled with vegetables and served with sweet chili sauce.', 4.99, NULL, 1),
```

Comment:
nebi bilo lose da bar za neke imamo link na sliku...ili ako cemo downloadat slike onda i ne moras

Reply:
To sam se cuo sa Marinom da cemo kasnije